VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -80,20 +75,16 @@ export abstract class WaitingRoomBaseDirective implements AfterContentChecked {
     conferenceStartedBy: string;
     phoneNumber$: Observable<string>;
     hasCaseNameOverflowed = false;
+    featureFlags = FEATURE_FLAGS;
 
     divTrapId: string;
 
-    panelTypes = ['Participants', 'Chat'];
-    panelStates = {
-        Participants: true,
-        Chat: false
-    };
-
     CALL_TIMEOUT = 31000; // 31 seconds
     callbackTimeout: ReturnType<typeof setTimeout> | number;
 
     loggedInUser: LoggedParticipantResponse;
     contactDetails = vhContactDetails;
+    subscriptions: Subscription[] = [];
```

Comment:
the component has examples of using the `takeUntil` operator. I would recommend that approach for consistency and ease of management

Reply:
updated, thanks